infra/feast-operator/internal/controller/featurestore_controller_db_store_test.go

```diff
@@ -353,39 +328,7 @@ var _ = Describe("FeatureStore Controller - db storage services", func() {
 			})
 			Expect(err).To(HaveOccurred())
 
-			Expect(err.Error()).To(Equal("secret key cassandra in secret online-store-secret contains invalid tag named type"))
-
-			By("Referring to a secret that contains parameter named registry_type")
-			resource = &feastdevv1alpha1.FeatureStore{}
-			err = k8sClient.Get(ctx, typeNamespacedName, resource)
-			Expect(err).NotTo(HaveOccurred())
-
-			secret = &corev1.Secret{}
-			err = k8sClient.Get(ctx, onlineSecretNamespacedName, secret)
-			Expect(err).NotTo(HaveOccurred())
-			secret.Data[string(services.OnlineDBPersistenceCassandraConfigType)] = []byte(cassandraYamlString)
-			Expect(k8sClient.Update(ctx, secret)).To(Succeed())
-
-			secret = &corev1.Secret{}
-			err = k8sClient.Get(ctx, registrySecretNamespacedName, secret)
-			Expect(err).NotTo(HaveOccurred())
-			secret.Data["sql_custom_registry_key"] = nil
-			secret.Data[string(services.RegistryDBPersistenceSQLConfigType)] = []byte(invalidSecretRegistryTypeYamlString)
-			Expect(k8sClient.Update(ctx, secret)).To(Succeed())
-
-			resource.Spec.Services.Registry.Local.Persistence.DBPersistence.SecretRef = corev1.LocalObjectReference{Name: "registry-store-secret"}
-			resource.Spec.Services.Registry.Local.Persistence.DBPersistence.SecretKeyName = ""
-			Expect(k8sClient.Update(ctx, resource)).To(Succeed())
-			resource = &feastdevv1alpha1.FeatureStore{}
-			err = k8sClient.Get(ctx, typeNamespacedName, resource)
-			Expect(err).NotTo(HaveOccurred())
-
-			_, err = controllerReconciler.Reconcile(ctx, reconcile.Request{
-				NamespacedName: typeNamespacedName,
-			})
-			Expect(err).To(HaveOccurred())
-
-			Expect(err.Error()).To(Equal("secret key sql in secret registry-store-secret contains invalid tag named registry_type"))
+			Expect(err.Error()).To(Equal("secret key cassandra in secret online-store-secret contains tag named type with value wrong"))
```

Comment:
nit ... can this error message be modified?
```suggestion
			Expect(err.Error()).To(Equal("secret key cassandra in secret online-store-secret contains tag named type with wrong value"))
```

Reply:
nm, disregard